### PR TITLE
Add a function to kill service on source/destination host

### PR DESCRIPTION
**Pull request by cliping**
Signed-off-by: lcheng <lcheng@redhat.com>

### CODE REVIEW

**Comment by cliping** [comment]
Test result: https://github.com/autotest/tp-libvirt/pull/3764

**Review by Yingshun — APPROVED** [review]
LGTM

File: virttest/utils_misc.py [hunk 1]
```diff
@@ -4313,3 +4314,22 @@ def _md5(fd):
         if _md5(fd_a) == _md5(fd_b):
             return True
     return False
+
+
+def kill_service(params):
+    """
+    Kill service on source or target host
+    :param params: Dictionary with the test parameters
```
Comment (dzhengfy) [inline virttest/utils_misc.py:4322]: Add a blank line above this

Comment (cliping) [reply virttest/utils_misc.py:4323]: Updated, thanks.

**Review by dzhengfy — APPROVED** [review]
LGTM

**Pull request merged**